engine-precompiles/src/lib.rs

```diff
@@ -47,7 +49,8 @@ impl From<PrecompileOutput> for evm::executor::PrecompileOutput {
     fn from(output: PrecompileOutput) -> Self {
         evm::executor::PrecompileOutput {
             exit_status: ExitSucceed::Returned,
-            cost: output.cost,
+            // This shouldn't fail, and is expected.
```

Comment:
What's this comment about? `into_u64` is infallible right?

Reply:
Yeah, that comment should go. Just an artifact from when I was using a different structure.